### PR TITLE
set up landing

### PR DESCRIPTION
**Pull request by tyeung2017**
relates #3 #11

### CODE REVIEW

**Review by KatiaAsh — APPROVED** [review]
:tea:

File: landing.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<head>
+  <meta charset="UTF-8">
+  <title>balloons SVG</title>
+  <link rel="stylesheet" href="landing.css">
+</head>
+
+<body>
+  <!-- <svg>
+    <polygon points="240,300 250,290 260,300" fill="gold"/>
+  </svg> -->
```
Comment (Wyrd00) [inline landing.html]: why is this in the landing page?

**Review by KatiaAsh — APPROVED** [review]

File: landing.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>balloons SVG</title>
+  <link rel="stylesheet" href="landing.css">
+</head>
+
+<body>
+  <div id="pop" class="disappear"></div>
+  <button>click me</button>
```
Comment (Wyrd00) [inline landing.html]: what about these?

**Review by KatiaAsh — APPROVED** [review]
:cool:

**Review by Wyrd00 — APPROVED** [review]
👍  sweeeeeeet

**Comment by tyeung2017** [comment]
relates #9

**Pull request merged**